addons/website/views/snippets/s_numbers_showcase.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_numbers_showcase" name="Numbers Showcase">
+    <section class="s_numbers_showcase pt80 pb80 o_colored_level">
+        <div class="container">
+            <div class="row o_grid_mode" data-row-count="8" style="gap: 24px;">
+                <div class="o_grid_item g-height-6 g-col-lg-5 col-lg-5 justify-content-start" style="--grid-item-padding-y: 32px; --grid-item-padding-x: 32px; grid-area: 1 / 1 / 9 / 7; z-index: 1;">
+                    <h2>
+                        <span class="display-2">Explore our<br/> key statistics</span>
+                    </h2>
```

Comment:
I can't properly style this title, probably because you defined the class in a span?

https://github.com/user-attachments/assets/603628ea-e523-40fb-8904-f965fe463957

Also, I think that the mockup has been a bit misleading... since this is not an "Intro" snippet but a "Content" one, the suggested heading size should be much smaller

| this PR | proposal |
|--------|--------|
| ![image](https://github.com/user-attachments/assets/185017d4-30a6-41ba-b7e5-974610e7eeda) | ![image](https://github.com/user-attachments/assets/832f2b9e-5504-42a9-8db6-bc62862e3365)|